### PR TITLE
Replaced "machined" label with "machine" #32373

### PR DESCRIPTION
**Pull request by Chillthrower**

### CODE REVIEW

**Review by mrc0mmand — APPROVED** [review]

**Comment by mrc0mmand** [comment]
I relabeled the existing issues/PRs in the background and dropped the machined label.

File: .github/advanced-issue-labeler.yml [hunk 1]
```diff
@@ -67,8 +67,8 @@ policy:
           - name: login
             keys: ['systemd-logind', 'loginctl', 'pam_systemd']
 
-          - name: machined
-            keys: ['systemd-machined', 'machinectl']
+          - name: machine
+            keys: ['systemd-machine', 'machinectl']
```
Comment (YHNdnzj) [inline .github/advanced-issue-labeler.yml:71]: Hmm? Why change this?? The name of the component is systemd-machined

**Review by YHNdnzj — CHANGES_REQUESTED** [review]

File: .github/advanced-issue-labeler.yml (hunk 1 above)
Comment (mrc0mmand) [reply .github/advanced-issue-labeler.yml:71]: Ugh, I completely overlooked this, sorry about that.

Comment (Chillthrower) [reply .github/advanced-issue-labeler.yml:71]: hey i just made the changes you have told i just made a pr for that

Comment (mrc0mmand) [reply .github/advanced-issue-labeler.yml:71]: -> https://github.com/systemd/systemd/pull/32378

**Pull request merged**